ingestion_server/ingestion_server/static/mock_sensitive_terms.txt

```diff
@@ -0,0 +1,3 @@
+running
+water
+bird
```

Comment:
I chose these because they match terms in our sample data. We can change these to anything that matches our sample data, if folks want.